src/components/application_manager/include/application_manager/commands/command_impl.h

```diff
@@ -138,6 +138,14 @@ class CommandImpl : public Command {
   */
   void SetAllowedToTerminate(const bool allowed) OVERRIDE;
 
+  /**
+   * @brief Check syntax of string from mobile
```

Comment:
@ZhdanovP Does this method checks only mobile messages?

Reply:
@AByzhynar yes, now only from mobile